readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -32,6 +32,11 @@ function create_ad_placement() {
     var offset;
 
     if ($(EXPLICIT_PLACEMENT_SELECTOR).length > 0) {
+        $(EXPLICIT_PLACEMENT_SELECTOR).attr("data-ea-publisher", "readthedocs");
```

Comment:
Do we not need this to respect the value coming back from the sustainability API?

Reply:
That will set it later.

Reply:
Ah, reading through the code, sounds like that's set dynamically after the call returns.

Reply:
Cool, should get this merged for deploy 👍